Document roadmap and dev process (#607)
BeaconCMS/beacon: CONTRIBUTING.md

# Contributing to Beacon

## Guidelines

* Issues aligned with our [roadmap](https://github.com/BeaconCMS/beacon_live_admin/blob/main/ROADMAP.md) have higher priority
  and those are [labeled as roadmap](https://github.com/BeaconCMS/beacon/labels/roadmap)
* [Bugs](https://github.com/BeaconCMS/beacon/labels/bug) tend to have higher priority unless they are of minor impact

> * If unsure where to start, look for issues tagged as [good first issue](https://github.com/BeaconCMS/beacon/labels/good%20first%20issue) to find good candidates of issues to start contributing

Reviewer: ```suggestion
* If unsure where to start, look for issues tagged as [good first issue](https://github.com/BeaconCMS/beacon/labels/good%20first%20issue) to find good candidates for your first contribution
```

> * Issues tagged [help wanted](https://github.com/BeaconCMS/beacon/labels/help%20wanted) are also good candidates but may require more knowledge of the project and time

Reviewer: ```suggestion
* Issues tagged [help wanted](https://github.com/BeaconCMS/beacon/labels/help%20wanted) are also good candidates but may require more project knowledge and/or time investment
```

## Found a bug?

> Please open a [bug reporet](https://github.com/BeaconCMS/beacon/issues/new?template=bug_report.md&labes=bug).

Reviewer 2: s/reporet/report/

If your pull-request addresses an issue then please add the corresponding
issue's number to the description of your pull-request.

## Prioritization

Issues tagged as [roadmap](https://github.com/BeaconCMS/beacon/labels/roadmap) indicate the work that is planned to be worked

> next on the short term, those have more priority over the other issues and will be included in the next releases.

Reviewer 2: Start a new sentence at "those have more priority".

> Issues tagged as [roadmap](https://github.com/BeaconCMS/beacon/labels/roadmap) indicate the work that is planned to be worked
> next on the short term. Those have more priority over the other issues and will be included in the next releases.

Reviewer: ```suggestion
Issues tagged as [roadmap](https://github.com/BeaconCMS/beacon/labels/roadmap) indicate the work that is planned to be done
next in the short term. Those have more priority over the other issues and will be included in the next releases.
```

Engaging on [feature request issues](https://github.com/BeaconCMS/beacon/issues?q=is%3Aissue+is%3Aopen+label%3Afeature-request) also helps us to prioritize the work,

> so feel free to add comments or discusson on feature request you'd like to see implemented and those will have more priority over the other issues.

Reviewer 2: s/discusson/discussion/
s/on feature request/on feature requests/

> Finally keep in mind that we can't guarantee which exact issues will be included in the very next release and we may give more
> priority to specific issues as decided by the core team as needed.

Reviewer: ```suggestion
Finally keep in mind that we can't guarantee which exact issues will be included in the very next release; the core team may
give more priority to specific issues as needed.
```
